target/linux/ramips/dts/RAK633.dts

```diff
@@ -0,0 +1,97 @@
+/dts-v1/;
+
+#include "mt7628an.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "rakwireless,rak633", "mediatek,mt7628an-soc";
+	model = "RAKwireless RAK633";
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x4000000>;
+	};
+	
+	leds {
+		compatible = "gpio-leds";
+			status {
+				label = "rak633:blue:status";
+				gpios = <&gpio1 12 GPIO_ACTIVE_LOW>;
+				};
+		};
+};
+
+&pinctrl {
+	state_default: pinctrl0 {
+		gpio {
+			ralink,group = "wled_an", "refclk", "wdt";
+			ralink,function = "gpio";
+		};
+	};
+};
+
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {

```

Comment:
@eikepedia Could you please change this to generic name `flash` as suggested by [DTS Checklist](https://openwrt.org/submitting-patches#dts_checklist) and check if it still boots?
>  5. The name of a node should reflect the function of the device and not its model. Examples for generic node names can be found in Section 2.2.2 Generic Names Recommendation